maggy/util.py

```diff
@@ -234,3 +229,48 @@ def _validate_ml_id(app_id, run_id):
     if prev_app_id == app_id and int(prev_run_id) >= run_id:
         return app_id, (int(prev_run_id) + 1)
     return app_id, run_id
+
+
+def _find_spark(conf=None):
+    """
+    Returns: SparkSession
+    """
+    # sp = SparkSession.builder.getOrCreate()
+    # sp.stop()
+
+    conf = SparkConf()
+    conf.set("num-executors", "1")
```

Comment:
this is dangerous when you are on Hopsworks. Hopsworks sets the configuration for Spark. Why do you need to set it to 1?

Reply:
Spark configuration is still being modified.